.github/workflows/checks.yml

```diff
@@ -133,13 +133,22 @@ jobs:
   # ---------------------------------------------------------- #
 
   run-tests:
-    name: Tests
+    name: Tests (${{ matrix.python-version }})
```

Comment:
note: I've added displaying python version here because otherwise `is_pr_event` value will be shown as well (as below).
<img width="206" alt="image" src="https://github.com/user-attachments/assets/58f59624-9d81-423c-8068-56493c3218bf">
`is_pr_event` shows because it's the default behavior for matrix to show all variables of matrix combination.